src/pymovements/gaze/_utils/parsing.py

```diff
@@ -192,26 +218,52 @@ def get_pattern_keys(compiled_patterns: list[dict[str, Any]], pattern_key: str)
 
 
 def parse_eyelink_event_start(line: str) -> str | None:
-    """Check if the line contains the start of an event and return the event name."""
-    if FIXATION_START_REGEX.match(line):
-        return 'fixation'
-    if SACCADE_START_REGEX.match(line):
-        return 'saccade'
-    if BLINK_START_REGEX.match(line):
-        return 'blink'
+    """Check if the line contains the start of an event and return the event name.
+
+    Returns event names with eye suffixes: e.g. 'fixation_left' or 'saccade_right'.
+    """
+    if match := FIXATION_START_REGEX.match(line):
+        eye = match.group('eye').upper()
+        suffix = '_left' if eye == 'L' else '_right'
+        return f'fixation{suffix}'
```

Comment:
I reconsidered this a bit. Maybe it would be better to put this information into a separate events column named `eye`.

This way it will be much easier to process later on.

So I would propose to change the function signature to:

```python
def parse_eyelink_event_start(line: str) -> tuple[str, str] | None:
```

If an event was parsed, the returned value would then be a tuple of the event name (`fixation`, `saccade`, `blink`) and the respective eye ( `left`, `right`).